### ISSUE TITLE
Should we install rtt, log4cpp and ocl independently, or install orocos-toolchain as a whole instead?

### ISSUE THREAD

**Issue by qkx515** [orocos-toolchain/rtt#272]
I want to install orocos-toolchain as a whole. So I went to [https://orocos-toolchain.github.io/](https://orocos-toolchain.github.io/).

And I saw there is only 

- Installing the RTT (pdf)

So I installed rtt, and I succeed. However, I found that I don't have ocl package.
```
# orocreate-pkg HelloWorld
orocreate-pkg: command not found
```
So I goes on  to install ocl. I saw this on the [https://github.com/orocos-toolchain/ocl/blob/toolchain-2.9/INSTALL](https://github.com/orocos-toolchain/ocl/blob/toolchain-2.9/INSTALL) .
```
NOTE - The Orocos Component Library is now part of the 
Orocos Toolchain and should seldomly be installed
separately. Install the Orocos Toolchain instead.
```
However, there is really nothing in this repo. I can only find orocos-toolchain 2.6 from [http://www.orocos.org/toolchain](http://www.orocos.org/toolchain). 

In addition, I got really confused that on another branch [https://github.com/orocos-toolchain/orocos_toolchain/tree/installation-script](https://github.com/orocos-toolchain/orocos_toolchain/tree/installation-script)
```
The Open RObot COntrol Software (Orocos) Toolchain is a bundle of multiple packages, which need to be build and installed separately.
- Orocos Real-Time Toolkit (rtt) - a component framework that allows us to write real-time components in C++
- Orocos Log4cpp (log4cpp) - a patched version of the Log4cpp library for flexible logging to files, syslog, IDSA and other destinations
- Orocos Component Library (ocl) - the necessary components to start an application and interact with it at run-time
```
---
**To sum it up, how can we install orocos-toolchain 2.9.0? Separately or As a whole?**

Also could any one change the default branch to the newest one? These branches are somewhat confusing. Thanks! :-)

### The same issues as https://github.com/orocos-toolchain/orocos_toolchain/issues/30

**Comment by roboticsai** [comment]
[Here](https://github.com/orocos/rtt_ros_integration/blob/hydro-devel/README.md) is one method to do so but it is integrated with ros.

**Comment by qkx515** [comment]
Thanks! 

Is it a good practice to use rtt_ros_integration for all the time? Feels like this one is more up-to-date and easy to use.
